inference/core/interfaces/http/http_api.py

```diff
@@ -572,6 +573,45 @@ async def clip_compare(
                         trackUsage(clip_model_id, actor, n=2)
                     return response
 
+            if CORE_MODEL_GROUNDINGDINO_ENABLED:
```

Comment:
Do you have some local un-pushed changes? Looks like this is left over copy+paste from the CLIP routes?